src/sass/layout/_statistics.scss

```diff
@@ -0,0 +1,33 @@
+.statistics {
+  min-height: 712.5px;
+  background-color: white;
+
+  @include center-inside-elem-flex;
+}
+
+.statistics__content {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 10px;
+  justify-content: space-between;
+  min-height: 526px;
+}
+
+.statistics__heading {
+  width: 506px;
+  height: 187px;
+  margin-top: 145px;
+
+  &::before {
```

Comment:
Why you decided to use content instead of just simply putting that text inside  ```<h2>```  tag? if you have it as content you cannot select any text from it, this is not really needed here, the main reason why we keep following the semantincs for static sites are for SEO, content is not being indexed by search enginers - https://www.searchcandy.uk/seo/technical-seo/css-pseudo-elements/

Reply:
Sounds good, ok